migrations/versions/25e44a14ecb8_added_psi_enabled.py

```diff
@@ -0,0 +1,34 @@
+"""added psi enabled
+
+Revision ID: 25e44a14ecb8
+Revises: 290a91f4eff6
+Create Date: 2022-09-22 15:52:08.936706
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '25e44a14ecb8'
+down_revision = '290a91f4eff6'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('performance_profile', 'report_date',
```

Comment:
Why this is needed for PSI enabled change?

Reply:
This is auto-generated, shall I remove the upgrade func.?

Reply:
Did you mean below line is auto-generated in the migration?

```
op.alter_column('performance_profile', 'report_date',
               existing_type=postgresql.TIMESTAMP(timezone=True),
               nullable=True)
```

Reply:
@kgaikwad Yes, that is correct, was present when migration file was generated.

Reply:
@patilsuraj767 They have added a disclaimer as well, he he
```
def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
```
I'll check and refactor, thanks.

Reply:
@kgaikwad @patilsuraj767 There is a new migration file now, changes pushed